johnsonandjohnson/Bodiless-JS
# feat(search): component library search component

## Changes


- Create wrapper class on Lunr in-browser search library
- Create SearchTool for generating search index.
- Search related components for site search feature.

## Test Instructions


- Edit search configures in site .env.site file
- Deploy test-site and review the search function.
- Add test content to pages, rebuild static site. Search test content.
- Check AC 1-4 of original issue.

## Related Issues

fixes #204
